index.html

```diff
@@ -34,6 +35,159 @@ <h2>I am a Software Developer</h2>
+          has been the industry's standard
+        </p>
+        <ul>
+          <li>html</li>
+          <li>Bootstrap</li>
+          <li>Ruby</li>
+        </ul>
+        <button type="button">
+          See Project
+        </button>
+      </div>
+      <div class="card card-with-bgimage">
+        <h2>Profesional Art Printing Data</h2>
+        <p>
+          A daily selection of privately personalized reads; no accounts or sign-ups required. 
+          has been the industry's standard
+        </p>
+        <ul>
+          <li>html</li>
+          <li>Bootstrap</li>
+          <li>Ruby</li>
+        </ul>
+        <button type="button">
+          See Project
+        </button>
+      </div>
+      <div class="card card-with-bgimage">
+        <h2>Profesional Art Printing Data</h2>
+        <p>
+          A daily selection of privately personalized reads; no accounts or sign-ups required. 
+          has been the industry's standard
+        </p>
+        <ul>
+          <li>html</li>
+          <li>Bootstrap</li>
+          <li>Ruby</li>
+        </ul>
+        <button type="button">
+          See Project
+        </button>
+      </div>
+      <div class="card card-with-bgimage">
+        <h2>Profesional Art Printing Data</h2>
+        <p>
+          A daily selection of privately personalized reads; no accounts or sign-ups required. 
+          has been the industry's standard
+        </p>
+        <ul>
+          <li>html</li>
+          <li>Bootstrap</li>
+          <li>Ruby</li>
+        </ul>
+        <button type="button">
+          See Project
+        </button>
+      </div>
+      <div class="card card-with-bgimage">
+        <h2>Profesional Art Printing Data</h2>
+        <p>
+          A daily selection of privately personalized reads; no accounts or sign-ups required. 
+          has been the industry's standard
+        </p>
+        <ul>
+          <li>html</li>
+          <li>Bootstrap</li>
+          <li>Ruby</li>
+        </ul>
+        <button type="button">
+          See Project
+        </button>
+      </div>
+      <div class="card card-with-bgimage">
+        <h2>Profesional Art Printing Data</h2>
+        <p>
+          A daily selection of privately personalized reads; no accounts or sign-ups required. 
+          has been the industry's standard
+        </p>
+        <ul>
+          <li>html</li>
+          <li>Bootstrap</li>
+          <li>Ruby</li>
+        </ul>
+        <button type="button">
+          See Project
+        </button>
+      </div>
+
+    </div>
+  </section>
+
+  <section class="about-me">
+    <h1>About me</h1>
+    <p>
+      Hello I’m a software developer! I can help you build a product , 
+      feature or website Look through some of my work and experience! 
+      If you like what you see and have a project you need coded, 
+      don’t hestiate to contact me.
+    </p>
+    <button type="button">Get My Resume</button>
+    <hr>
```

Comment:
- [ ] Please, consider the button styling for the "Get My Resume" button: Active, Hover, and Pressed
![2022-03-26 20 45 58](https://user-images.githubusercontent.com/49064106/160249539-d6ca7e03-0a65-4431-b1a9-b59477591e82.gif)